src/mca/rmaps/round_robin/rmaps_rr_mappers.c

```diff
@@ -720,11 +732,14 @@ int prte_rmaps_rr_byobj(prte_job_t *jdata, prte_app_context_t *app,
                     if (PRTE_ERR_TAKE_NEXT_OPTION == rc) {
                         /* move to next node */
                         nodefull = true;
+                        PMIX_RELEASE(proc);
                         break;
                     } else if (PRTE_SUCCESS != rc) {
                         /* got an error */
+                        PMIX_RELEASE(proc);
                         goto errout;
                     }
+                    PMIX_RELEASE(proc);
```

Comment:
Not sure I understand this release. The `proc` object gets instantiated and added to the `jdata` object on line 724. If we release it here, then the information in `jdata` is now stale as the object it points to will have been released. Are you sure this release should be here?